publish-release.sh

```diff
@@ -30,40 +31,52 @@ EOD
 }
 
 # get the options
-while getopts h c ; do
+while getopts hm c ; do
     case $c in
 	    h) usage ; exit 0 ;;
+            m) minimal=true ;;
 	    \?) usage ; exit 2 ;;
     esac
 done
 shift `expr $OPTIND - 1`
 
-if [ $# -ne 1 ] ; then
+if [ $# -gt 1 ] ; then
     usage
     exit 2
 fi
 
-releasetag="$1"
-export OP_VERSION="$releasetag"
+
+if [ $# -eq 1 ] ; then
+  releasetag="$1"
+  export OP_VERSION="$1"
+  message="Publish release"
+else
+  message="Publish version"
+fi
 
 . "$DIR/env.build.sh"
 
-make yaml yaml-ns-scoped
 $DIR/build.sh
+$DIR/push-image.sh
+make yaml yaml-ns-scoped
 
-echo "Update Qserv images in manifests/base/image.yaml"
-sed -ri  "s/^(\s*image: qserv\/.*:).*/\1$releasetag/" $DIR/manifests/base/image.yaml
-echo "Update release number in documentation"
-find $DIR/doc -type f -print0 | xargs -0 sed -ri  "s/RELEASE=\".*\"/RELEASE=\"$releasetag\"/"
-sed -ri  "s/RELEASE=\".*\"/RELEASE=\"$releasetag\"/" $DIR/README.md
+if [ $minimal = false ]; then
```

Comment:
I do not see `minimal` initialized, so its default value is an empty string and this statement will result in error (`$minimal` without double quotes will result in nothing, so it will be like `if [ = false ];`)